apps/language_server/test/providers/document_symbols_test.exs

```diff
@@ -1216,6 +1218,16 @@ defmodule ElixirLS.LanguageServer.Providers.DocumentSymbolsTest do
                     children: [],
                     kind: 5,
                     name: "@type my_with_args_when(key, value)"
+                  },
+                  %Protocol.DocumentSymbol{
+                    children: [],
+                    kind: 5,
```

Comment:
man, I really dislike these magic numbers...
one idea is to use the experimental protocol structs. Another is to define functions in `Protocol.DocumentSymbol` named for the kinds that fill them in for you. i.e.
```elixir
Protocol.DocumentSymbol.class(name: "@type abc")
```

Reply:
Extracted to a separate issue https://github.com/elixir-lsp/elixir-ls/issues/831